test/job/adapters/slurm_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class TestSlurm < Minitest::Test
+  include TestHelper
+
+  def slurm_instance(config = {})
+    OodCore::Job::Factory.build({ adapter: 'slurm' }.merge(config))
+  end
+
+  def test_submit_interface
+    slurm = slurm_instance
+
+    assert(slurm.respond_to?(:submit))
+    veryify_keywords(slurm, :submit, [:after, :afterok, :afternotok, :afterany])
+    verify_args(slurm, :submit, 1)
```

Comment:
This replaces this line from rspec. Minitest doesn't have the same facility, but I've added some helper methods to do the same.

https://github.com/OSC/ood_core/blob/65f8ac5dcebf6b4546d1082c308b724c26e2eca3/spec/job/adapters/slurm_spec.rb#L11